### PR TITLE
Update README to include Approach

### PR DESCRIPTION
**Pull request by adj7388**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,3 +197,86 @@ Make sure you have a proper [datadog agent](http://docs.datadoghq.com/guides/bas
 Next, [get your datadog API key](https://app.datadoghq.com/account/settings#api). Modify your service script (if you created one) to export the variable `DATADOG_API_KEY` and set it to your newly created key.
 
 Once you restart your service, and have traffic coming in, you should now have some basic stats in your datadog account that you can now report. These stats start with the key of `node.express`.
+
+## Approach
+
+_See our [Approach Criteria Evidence](APPROACH_EVIDENCE.md)_
+
+**Initial status:**
```
Comment (poorgeek) [inline README.md]: Should be a header and not just bold

File: README.md [hunk 2]
```diff
@@ -198,3 +197,86 @@ Make sure you have a proper [datadog agent](http://docs.datadoghq.com/guides/bas
...
+* Publish a project [Project Road Map](Project-Road-Map) with our initial release specified as our minimum viable product.
+
+## Daily Scrum
+Start iterative development process. Given the compressed time frame:
+* Use continuous deployment so the latest code would always be on the prototype server, so planning specific releases became less important
+* Use a more flexible iteration process: First iteration would include user stories for a minimum viable product. 
+  * took responsibility for producing an MVP by the submission date.
+  * subsequent iterations could be planned during daily scrums to add additional features from our release plan and road map
+
+## Acceptance Testing
```
Comment (poorgeek) [inline README.md]: This section should mention that the PO is performing acceptance testing while receiving additional feedback from users.

File: README.md [hunk 3]
```diff
@@ -198,3 +197,86 @@ Make sure you have a proper [datadog agent](http://docs.datadoghq.com/guides/bas
...
+  * Set scrum time
+  * Set up code repository, issue tracker, project wiki
+* Conduct project kickoff meeting with the client (in this case our proxy users)
+* Create Risk Management plan, to be updated
+  * during and after requirements elicitation with the customer
+  * during release and iteration planning meetings
+  * during retrospectives
+
+### Initial Requirements Elicitation and Design Session
+* Craft a [vision](Project-Charter) for the project with users
```
Comment (poorgeek) [inline README.md]: Update link to point to the wiki

File: README.md [hunk 4]
```diff
@@ -198,3 +197,86 @@ Make sure you have a proper [datadog agent](http://docs.datadoghq.com/guides/bas
...
+  * update stories or add new ones as required
+* prioritize stories according to user
+
+### Release Planning/Iteration Planning
+With user stories completed and prioritized, conduct initial release planning meeting:
+* Add estimates to each story
+* Group the stories into multiple releases, based on users priorities and developer time estimates  
+* Break user stories into tasks
+* Assign stories to developers for the first iteration
+* Publish a project [Project Road Map](Project-Road-Map) with our initial release specified as our minimum viable product.
```
Comment (poorgeek) [inline README.md]: Update link to point to the wiki

File: README.md (hunk 2 above)
Comment (adj7388) [inline README.md]: You mean under the Acceptance Testing heading? Or in one of those other headings? The first line under Acceptance Testing is essentially what you are saying.

Comment (poorgeek) [inline README.md]: I think we need to state that true acceptance is in the hands of the PO. Users can say what they want, but it's up to the PO to decide if all of the acceptance criteria have be met and if the story can be closed.

Comment (adj7388) [inline README.md]: OK gotcha

**Pull request merged**